charts/generic-prometheus-alerts/values.yaml

```diff
@@ -10,7 +10,13 @@ targetApplication: ""
 ingressAlertsEnabled: true
 
 # The alert severity label is used within Alert Manager and determines which slack channel alerts are sent to.
-alertSeverity: digital-prison-service-dev
+alertSeverity: ""
+
+# These values used in the rule labels, which enables better filtering of alerts in alertmanager.  Global helm values used by these labels are passed in at helm upgrade/install time.
+businessUnit: hmpps
+# global:
+#   environment: "xxx"
+#   productId: "xxx"
```

Comment:
worth updating the [README](https://github.com/ministryofjustice/hmpps-helm-charts/blob/main/charts/generic-prometheus-alerts/README.md)?